src/nft/pages/collection/index.tsx

```diff
@@ -1,5 +1,30 @@
-export const Collection = () => {
-  return <div>Collection Page</div>
+import { useWindowSize } from 'hooks/useWindowSize'
+import { useEffect } from 'react'
+import { useQuery } from 'react-query'
+import { useParams } from 'react-router-dom'
+
+import { useIsMobile } from '../../hooks/useIsMobile'
+import { CollectionStatsFetcher } from '../../queries'
+import { CollectionDesktop } from './CollectionDesktop'
+import { CollectionMobile } from './CollectionMobile'
+
+const Collection = () => {
+  const { contractAddress } = useParams()
+
+  const { isMobile, setMobileWidth } = useIsMobile()
+
+  const { data: collectionStats } = useQuery(['collectionStats', contractAddress], () =>
+    CollectionStatsFetcher(contractAddress as string)
+  )
+
+  const { width } = useWindowSize()
+  useEffect(() => {
+    setMobileWidth(width ?? 0)
```

Comment:
I'm surprised this isn't handled at a higher level, considering that it's global state.
Is this set anywhere else? Does it _need_ to be set here? _Could_ it be set at a higher level, to avoid duplicating this logic in components that shouldn't include this logic?

Reply:
updated the isMobile hook